Modules/Loadable/SubjectHierarchy/Widgets/qSlicerSubjectHierarchyPluginHandler.cxx

```diff
@@ -379,6 +384,23 @@ qSlicerSubjectHierarchyAbstractPlugin* qSlicerSubjectHierarchyPluginHandler::sel
   return candidatePlugins[0];
 }
 
+//----------------------------------------------------------------------------
+void qSlicerSubjectHierarchyPluginHandler::observeSubjectHierarchyNode(vtkMRMLSubjectHierarchyNode* shNode)
```

Comment:
I suggest to rename the function to `AddSubjectHierarchyNodeObservers`

Reply:
Happy too. @cpinter ?

Reply:
I don't mind. But please have it start with lower case, because it's a Qt class.
Btw my opinion is that since this is integrated, creating another PR would be needed, which I think is unreasonable for such a change. The name of the function is not misleading, and it does not really do against conventions, because again, this is not a VTK class.

Reply:
> because again, this is not a VTK class.

Sorry for the noise, you are right there is no need to change it.

Reply:
Thanks, @jcfr no problem